app/src/main/res/layout/topic_train_fragment.xml

```diff
@@ -26,35 +27,48 @@
       <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:orientation="vertical"
-        android:padding="32dp">
+        android:orientation="vertical">
+
         <TextView
           android:id="@+id/master_skills_text_view"
           android:layout_width="wrap_content"
           android:layout_height="wrap_content"
+          android:fontFamily="@font/roboto_medium"
           android:text="@string/topic_train_master_these_skills"
           android:textColor="@color/oppiaPrimaryText"
-          android:textSize="20sp"/>
+          android:layout_marginTop="@dimen/margin_48dp"
+          android:layout_marginStart="@dimen/margin_28dp"
+          android:layout_marginEnd="@dimen/margin_28dp"
+          android:textSize="20sp" />
+
         <TextView
           android:id="@+id/skills_description_text_view"
           android:layout_width="wrap_content"
           android:layout_height="wrap_content"
-          android:layout_marginTop="16dp"
+          android:layout_marginTop="@dimen/margin_8dp"
+          android:layout_marginStart="@dimen/margin_28dp"
+          android:layout_marginEnd="@dimen/margin_28dp"
+          android:fontFamily="@font/roboto_regular"
           android:text="@string/topic_train_skills_description"
           android:textColor="@color/oppiaSecondaryText"
-          android:textSize="16sp"/>
+          android:textSize="16sp" />
         <androidx.recyclerview.widget.RecyclerView
           android:id="@+id/skill_recycler_view"
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
-          android:layout_marginTop="16dp"
+          android:layout_marginTop="@dimen/margin_20dp"
+          android:layout_marginStart="@dimen/margin_28dp"
+          android:layout_marginEnd="@dimen/margin_28dp"
           app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"/>
         <Button
+          android:layout_width="160dp"
+          android:layout_height="50dp"
```

Comment:
Similarly, we should be using multiples of 4. Please check with Chantel on what the value should be here.

Reply:
@mschanteltc please take a look at the size specified for `START` button over here, as per the mock width is 160px and height is 39px and we need to specify size as multiples of 4 as per material specification.

Reply:
@veena14cs I updated the mocks so that the height is 36 px. The button is spaced 40 px away from the bottom of the checklist.

Reply:
Thanks @mschanteltc. Done the changes.